src/packages/scalapb/update-hash.py

```
@@ -0,0 +1,62 @@
+def get_latest_release():
+    url = 'https://api.github.com/repos/scalapb/ScalaPB/releases/latest'
+    with urllib.request.urlopen(url) as response:
+        data = json.loads(response.read())
+    return data['tag_name'].lstrip('v')
+
+def get_jar_hash(version):
+    url = f"https://github.com/scalapb/ScalaPB/releases/download/v{version}/scalapbc-{version}.zip"
+    cmd = ['nix-prefetch-url', url]
+    result = subprocess.run(cmd, capture_output=True, text=True)
+    hash_legacy = result.stdout.strip().split('\n')[-1]
+    
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)